tests/main/hidden-snap-dir/task.yaml

```diff
@@ -120,7 +120,7 @@ execute: |
     "$NAME".echo "$data" > "$HOME/snap/$NAME/common/file"
 
     echo "Refresh the snap"
-    "$TESTSTOOLS"/snaps-state install-local "$NAME"
+    "$TESTSTOOLS"/snaps-state install-local "$NAME" --ignore-running
```

Comment:
why do we need --ignore-running here?

Reply:
Yeah, removed; it's not needed here and also in the refresh test; added it initially due to the confusion caused by leaking op-remove-retry test.